airgun/browser.py

```diff
@@ -274,256 +168,16 @@ def _get_remote_browser(self):
 
         Note: should not be called directly, use :meth:`get_browser` instead.
         """
-        self._webdriver = webdriver.Remote(
-            command_executor=settings.selenium.command_executor,
-            desired_capabilities=self._get_webdriver_capabilities(),
-        )
+        desired_capabilities = self.web_kaifuku['webdriver_options']['desired_capabilities']
+        desired_capabilities.update({'name': self.test_name})
```

Comment:
There should be a setter method for this property, rather than updating the dictionary by reference here.